openpaas-james/integration-tests/jmap/memory-jmap-integration-tests/src/test/java/com/linagora/openpaas/james/MemoryLinagoraKeystoreSetMethodTest.java

```diff
@@ -0,0 +1,23 @@
+package com.linagora.openpaas.james;
+
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.mailbox.inmemory.InMemoryId;
+import org.apache.james.modules.TestJMAPServerModule;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linagora.openpaas.james.app.MemoryServer;
+import com.linagora.openpaas.james.common.LinagoraKeystoreSetMethodContract;
+
+class MemoryLinagoraKeystoreSetMethodTest implements LinagoraKeystoreSetMethodContract {
+    @RegisterExtension
+    static JamesServerExtension jamesServerExtension = new JamesServerBuilder<>(JamesServerBuilder.defaultConfigurationProvider())
+        .server(configuration -> MemoryServer.createServer(configuration)
+            .overrideWith(new TestJMAPServerModule()))
+        .build();
+
+    @Override
+    public String generateAccountIdAsString() {
+        return "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6";
+    }
```

Comment:
Why do we need this?

Reply:
We need bob's account id? Or we gonna get an error from the request if the account id is wrong? I just took the pattern of other test suites TBH

Reply:
The accoutn id is a constant, can be hardcoded and do not depend of the underlying storage technology...

Reply:
Probably can do some refactorings on other test classes then